pkg/kdump/config-client.js

```diff
@@ -273,7 +278,7 @@ export class ConfigFile {
                 if ("core_collector" in settings._internal)
                     settings._internal.core_collector.value = settings._internal.core_collector.value + " -c";
                 else
-                    settings._internal.core_collector = { value: "makedumpfile -c" };
+                    settings._internal.core_collector = { value: "makedumpfile -l --message-level 7 -d 31 -c" };
```

Comment:
DRY -- can you put this as a constant at the top, with a comment where you got it from?

Reply:
ACK